packages/page-contracts/src/Contracts/Add.tsx

```diff
@@ -64,7 +64,8 @@ function Add ({ onClose }: Props): React.ReactElement {
   const isValid = isAddressValid && isNameValid && isAbiValid;
 
   return (
-    <Modal header={t('Add an existing contract')}>
+    <Modal header={t('Add an existing contract')}
+      onClose={onClose}>
```

Comment:
```suggestion
    <Modal
      header={t('Add an existing contract')}
      onClose={onClose}
    >
```

... eslint should really sort these out and format correctly.

Reply:
✅